modules/theme/scripts/build.js

```diff
@@ -177,14 +177,17 @@ const createSampleTheme = () => {
                 const files = fs.readdirSync(path.join(defaultThemePath, folder));
 
                 files.map((file) => {
-                    const fileNameSplit = file.split(".");
-                    const content = "/*******************************\n" +
-                        `     ${titleCase(fileNameSplit[0], "-")} ${titleCase(fileNameSplit[1], " ")}\n` +  
-                        "********************************\n";
-
-                    fs.writeFileSync(path.join(folderPath, file), content, (error) => {
-                        console.error(error);
-                    });
+
+                    if (file !== ".DS_Store") {
```

Comment:
.DS_Store files are already ignored right?

Reply:
It's been added when the theme gets build.